agent/create_delete_topics_test.go

```diff
@@ -0,0 +1,82 @@
+package agent
+
+import (
+	"testing"
+
+	"github.com/spirit-labs/tektite/kafkaprotocol"
+	"github.com/spirit-labs/tektite/objstore/dev"
+	"github.com/spirit-labs/tektite/transport"
+	"github.com/stretchr/testify/require"
+)
+
+func setupAgentWithoutTopics(t *testing.T, cfg Conf) (*Agent, *dev.InMemStore, func(t *testing.T)) {
```

Comment:
this method seems unnecessary - can just use the setupAgents method and pass in the topics you want

Reply:
`setupAgent` calls `setupTopics` which uses `control.Client.CreateTopic` directly. I wanted to test the topic creation using `conn.SendRequest`.
This is why, I created this function.